site/pages/docs/versions/v4.0/v4.0.30-en.hbs

```diff
@@ -7,121 +7,167 @@
-			<li>Mineur - Fils de syndication - Ajout de la lecture directe des fichiers XML qui sont dans la limite des CORS.</li>
-			<li>Correctif - Menu - Ouverture du menu mobile lorsque le menu est statique dans la page</li>
-			<li>Mineur - Fils de syndication - Lecture d'un fil d'actualité en format JSON similaire à ATOM XML.</li>
-			<li>Mineur - Fils de syndication - Retrait de la lecture d'un fil d'actualité via l'API Yahoo. Ceci est remplacé par une lecture directe.</li>
-			<li>Mineur - Tableaux - Retrait de la pagination lorsqu'il y a seulement une seule page de résultat</li>
-			<li>Mineur - Tableaux - Interface de filtrage, ajout l'option de filtrer les résultats avec des case à cocher</li>
-			<li>Correctif - Chronologie intégée Twitter - Retrait de l'example avec code obsolete de Twitter.</li>
-			<li>Correctif - Tableaux - Interface de filtrage - adjustement lorsqu'il y a plusieur case à cocher pour plus qu'une colonne.</li>
-			<li>Développeur de la BOEW - Modification du script de compilation et de ses dépendances.</li>
+			<li>Patch - Lightbox - Overwritten window when clicking on a current page link</li>
+			<li>Minor - Tables - Filter interface, adding support for <code>input[type=date]</code> when resetting</li>
+			<li>Minor - RSS feeds - Adding live XML file reading for files within the CORS limit</li>
+			<li>Patch - Menu - Opening the mobile menu when the menu is static in the page</li>
+			<li>Minor - RSS feeds - News feed reading in ATOM-like JSON XML format</li>
+			<li>Minor - RSS feeds - Retrieving the news feed reading by the Yahoo API. This is replaced by a live reading.</li>
+			<li>Minor - Tables - Removing page numbering when there is only one results page</li>
+			<li>Minor - Tables - Filter interface, adding the filter results option with check boxes</li>
+			<li>Patch - Integrated Twitter Timeline - Removing the example with the obsolete Twitter code</li>
+			<li>Patch - Tables - Filter interface, making adjustments when there are several check boxes in more than one column</li>
+			<li>WET developer - Modifying the compilation script and its dependencies</li>
 		</ul>
-
 		<p>Canada.ca (GCWeb):</p>
 		<ul>
-			<li><a href="https://wet-boew.github.io/themes-dist/GCWeb/gcweb-theme/release/v5.0-fr.html">Publication de la version 5.0 (Notes de versions)</a></li>
+			<li><a href="https://wet-boew.github.io/themes-dist/GCWeb/gcweb-theme/release/v5.0-en.html">Version 5.0 publication (Release notes)</a></li>
 		</ul>
 	</section>
 	<section>
-		<h3>Fureteur supporté</h3>
-		<p>Tel que défini par la <a hreflang="en" href="http://wet-boew.github.io/wet-boew-documentation/decision/2.html">Décision de conception 2: Fureteur supporté</a> (en anglais)</p>
-		<ul>
-			<li>Chrome 71</li>
-			<li>Chrome 70</li>
-			<li>Safari 12.0</li>
-			<li>Firefox 65</li>
-			<li>Firefox 64.0.2</li>
-			<li>Firefox ESR - 60.5.0</li>
-			<li>IE 11</li>
-			<li>Edge 18</li>
-		</ul>
+		<div lang="fr">
+			<h3>Fureteur supporté</h3>
```

Comment:
Can you use the corresponding English translation from the previous release notes 4.0.29?